src/TestInfrastructure/Extensions/FileSystemExtensions.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2021 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System.IO.Abstractions;
+using Moq;
+
+namespace SonarLint.VisualStudio.Integration.UnitTests.Extensions
```

Comment:
Is that the right namespace? I never noticed what's the default namespace in the tests infra project but this looks off.

Reply:
It does look weird. I'll leave it because it's the default. We can bulk-change the namespace later (it'll change a lot of product files too).